### PR TITLE
Another round of `test_create.py` refactoring

### PR DESCRIPTION
**Pull request by kenodegard**
### Description

Refactored tests:
- `test_create_only_deps_flag`
- `test_install_update_deps_flag`
- `test_install_only_deps_flag`
- `test_install_update_deps_only_deps_flags`
- `test_install_features`
- `test_clone_offline_simple`

Refactored and moved to `tests/cli/test_config.py`:
- `test_conda_config_describe`
- `test_conda_config_validate`
- `test_conda_config_validate_sslverify_truststore`

Xref https://github.com/conda/conda/issues/13292

### Checklist - did you ...

- [ ] ~Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?~
     
- [x] Add / update necessary tests?
- [ ] ~Add / update outdated documentation?~

### CODE REVIEW

File: conda/base/context.py [hunk 1]
```diff
@@ -165,7 +165,9 @@ def default_python_validation(value):
 
 def ssl_verify_validation(value):
     if isinstance(value, str):
-        if not value == "truststore" and not isfile(value) and not isdir(value):
+        if sys.version_info < (3, 10) and value == "truststore":
+            return "`ssl_verify: truststore` is only supported on Python 3.10 or later"
+        elif value != "truststore" and not exists(value):
```
Comment (jezdez) [inline conda/base/context.py:170]: Curious, but a good idea!

File: conda/testing/integration.py [hunk 2]
```diff
@@ -410,7 +410,7 @@ def tempdir():
 
 
 def reload_config(prefix):
-    prefix_condarc = join(prefix + os.sep, "condarc")
+    prefix_condarc = join(prefix, "condarc")
```
Comment (jezdez) [inline conda/testing/integration.py:413]: Odd

File: tests/cli/test_config.py [hunk 3]
```diff
@@ -722,3 +716,194 @@ def test_custom_multichannels_prepend_duplicate(conda_cli: CondaCLIFixture):
...
+
+def test_conda_config_describe(
+    tmp_env: TmpEnvFixture,
+    conda_cli: CondaCLIFixture,
+    monkeypatch: MonkeyPatch,
+):
+    RE_PARAMETERS = (
+        re.compile(rf"^# # {name} \(", flags=re.MULTILINE)
+        for category, names in context.category_map.items()
+        if category not in ("CLI-only", "Hidden and Undocumented")
```
Comment (jezdez) [inline tests/cli/test_config.py:729]: Not a huge fan of this hard-coding..

File: tests/test_create.py [hunk 4]
```diff
@@ -1250,105 +1250,136 @@ def test_create_no_deps_flag(test_recipes_channel: Path, tmp_env: TmpEnvFixture)
...
+    conda_cli: CondaCLIFixture,
+):
+    # ├─ dependent
+    # │  └─ dependency
+    # ├─ another_dependent
+    # │  └─ dependent
+    # │     └─ dependency
+    # └─ other_dependent
+    #    └─ dependent
+    #       └─ dependency
```
Comment (jezdez) [inline tests/test_create.py:1265]: Nice! 🌲

**Review by jezdez — APPROVED** [review]

File: tests/cli/test_config.py (hunk 3 above)
Comment (kenodegard) [reply tests/cli/test_config.py:729]: you're referring to `("CLI-only", "Hidden and Undocumented")`?

yeah I agree, I'm also not a fan of how we define or handle the condarc variable descriptions

**Pull request merged**